package.json

```diff
@@ -19,48 +19,50 @@
   "scripts": {
     "format": "prettier --write ./bin ./lib ./src ./test",
     "lint": "eslint ./bin/**.cjs ./lib/**.cjs ./src/**.js ./test/**.cjs ",
-    "test": "pnpm test:unit && tape './test/*.cjs'",
+    "test": "npm run test:unit && tape './test/*.cjs'",
     "test:unit": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
-    "demo": "cd ./test/demo && pnpm start",
+    "demo": "cd ./test/demo && npm start",
     "build": "esbuild src/index.js --bundle --minify --platform=node --outfile=./dist/index.cjs"
   },
+  "ManifestComments": [
+    "Pinned update-notifier to 5.1.0, v6+ requires ESM"
+  ],
   "devDependencies": {
-    "decompress-zip": "0.3.3",
-    "eol": "0.9.1",
-    "esbuild": "0.14.42",
-    "eslint": "8.16.0",
-    "eslint-plugin-jsdoc": "^39.3.2",
-    "jest": "28.1.0",
-    "jest-environment-jsdom": "28.1.0",
-    "jsdom": "19.0.0",
-    "nock": "13.2.4",
-    "pnpm": "7.1.7",
-    "prettier": "2.6.2",
-    "redtape": "1.0.0",
-    "tape": "5.5.3"
+    "decompress-zip": "^0.3.3",
+    "eol": "^0.9.1",
+    "esbuild": "^0.14.47",
+    "eslint": "^8.18.0",
+    "eslint-plugin-jsdoc": "^39.3.3",
+    "jest": "^28.1.1",
+    "jest-environment-jsdom": "^28.1.1",
+    "jsdom": "^20.0.0",
+    "nock": "^13.2.7",
+    "prettier": "^2.7.1",
+    "redtape": "^1.0.0",
+    "tape": "^5.5.3"
   },
   "dependencies": {
-    "archiver": "5.3.1",
-    "deprecate": "1.1.1",
-    "extract-zip": "2.0.1",
-    "graceful-fs-extra": "2.0.0",
-    "graceful-ncp": "3.0.0",
-    "inherits": "2.0.4",
-    "lodash": "4.17.21",
-    "plist": "3.0.5",
-    "progress": "2.0.3",
-    "rcedit": "3.0.1",
-    "recursive-readdir-sync": "1.0.6",
-    "request": "2.88.2",
-    "rimraf": "3.0.2",
-    "semver": "7.3.7",
-    "simple-glob": "0.2.0",
-    "tar-fs": "2.1.1",
-    "temp": "github:adam-lynch/node-temp#remove_tmpdir_dep",
-    "thenify": "3.3.1",
-    "update-notifier": "5.1.0",
-    "winresourcer": "0.9.0",
-    "yargs": "17.5.1"
+    "archiver": "^5.3.1",
+    "deprecate": "^1.1.1",
+    "extract-zip": "^2.0.1",
+    "graceful-fs-extra": "^2.0.0",
+    "graceful-ncp": "^3.0.0",
+    "inherits": "^2.0.4",
+    "lodash": "^4.17.21",
+    "plist": "^3.0.5",
+    "progress": "^2.0.3",
+    "rcedit": "^3.0.1",
+    "recursive-readdir-sync": "^1.0.6",
+    "request": "^2.88.2",
+    "rimraf": "^3.0.2",
+    "semver": "^7.3.7",
+    "simple-glob": "^0.2.0",
+    "tar-fs": "^2.1.1",
+    "temp": "^0.9.4",
```

Comment:
Not sure what the story is with this. My assumption is that @adam-lynch forked `temp`, made changes and submitted those as a PR. Then didn't want to wait for them to be merged, so pointed to his own fork. It looks like there is no code difference between his version and the official one now, except there have been many commits/new releases done on the official `temp`. I could be wrong on that. But after updating it, I didn't see any red flags go off (I ran all the npm scripts, and none failed).

Reply:
Sounds fine to me